sktime/forecasting/compose/_reduce.py

```diff
@@ -442,10 +443,39 @@ def _predict_last_window(
         return y_pred[fh_idx]
 
 
+class _SimpleReducer(_Reducer):
+    strategy = "simple"
+    _tags = {
+        "requires-fh-in-fit": False,  # is the forecasting horizon required in fit?
+    }
+
+    def __init__(self, regressor):
+        # storing hyper-parameters
+        self.regressor = regressor
+        super(_SimpleReducer, self).__init__()
+
+    def _fit(self, y, X=None, fh=None):
+        X = X.to_numpy()
+        y = y.to_numpy().reshape(-1, 1)
+        self.regressor_ = clone(self.regressor)
+        self.regressor_.fit(X, y)
+
+        return self
+
+    def _predict(self, fh, X=None, return_pred_int=False, alpha=None):
+        assert not return_pred_int
```

Comment:
Instead of having an assert statement here, we probably want to raise a more informative error message:
```python
if return_pred_int:
    raise NotImplementedError(f"{self.__class__.__name__} does not support prediction intervals.")
```